collect_app/src/androidTest/java/org/odk/collect/android/feature/instancemanagement/SendFinalizedFormTest.kt

```diff
@@ -150,8 +150,7 @@ class SendFinalizedFormTest {
 
             .clickSendFinalizedForm(2)
             .sortByDateNewestFirst()
-            .selectForm(0)
-            .selectForm(1)
+            .clickSelectAll()
```

Comment:
Here we wanted to make sure that the older form was being sent first and to do that we selected the newer one first. If we use the `select all` option it is not guaranteed right? Probably it works like that and selects forms starting from the top of the list but it's not obvious. Can we improve it somehow?

Reply:
I think this is a good compromise for now: reversing the order causes a fail, and it avoids using the problematic matcher. I definitely agree that selecting them individually provides a bit more coverage, but we'd need to write a new implementation of `selectForm` that I didn't want to spend time on vs other things.